test/test_alexa_app_customslot.js

```diff
@@ -10,6 +10,30 @@ chai.config.includeStack = true;
 import * as Alexa from '..';
 
 describe("Alexa", function() {
+
+  context("#alexa.request.slot", function () {
+    var mockRequest = mockHelper.load("intent_request_airport_info.json");
+    var testReq;
+    beforeEach(function () {
+      testReq = mockRequest;
+    });
+
+    afterEach(() => {
+      testReq = null;
+    });
+
+    it("get slot value if correct", function () {
+      var req = new Alexa.request(testReq);
+      expect(req.slot("AirportCode", "SEA")).to.eql("JFK");
+    });
+
+    it("gets defaultValue if slot value is undefined", function () {
+      testReq.request.intent.slots.AirportCode.value = undefined;
```

Comment:
If you make sure a change, note that `mockRequest` itself is modified (as Objects are by-ref), and if a new test will be written after this it will fail.
I suggest loading the JSON in the `beforeEach`, to ensure there will be no affect on other test cases.

Reply:
good point. updated.